frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -30,7 +32,7 @@ import {
   TLM_ENTITY_TYPE as TLM_ET,
   TLM_SUB_TYPE as TLM_ST,
   TracimComponent,
-  getCurrentContentVersionNumber
+  wrapMentionInSpanTag
```

Comment:
```suggestion
  wrapMentionsInSpanTags
```

There can be several mentions, right? A plural seems therefore more correct.

Reply:
Changed